test/services/adsService.test.js

```diff
@@ -305,6 +306,69 @@ describe('adsService', () => {
         })
       })
     })
+
+    describe('when there is two ads: one match and one not (because of gender)', () => {
+      const googleAdForFemale = new Ad('Google', 'Google image').active().forFemale().get()
+      const facebookAdForMale = new Ad('Facebook', 'Facebook image').active().forMale().get()
+
+      before(() => {
+        const ads = {
+          [facebookAdForMale.id]: facebookAdForMale,
+          [googleAdForFemale.id]: googleAdForFemale
+        }
+        Database('ads').set(ads)
+      })
+
+      for (let i = 0; i < 10; i++) {
```

Comment:
No era que había una forma de hacer que se ejecute `n` veces más allá de un `for`? jajaj

Reply:
si, esa era la forma 👍